modules/ROOT/pages/bat-execute-task.adoc

```diff
@@ -1,58 +1,32 @@
-= Execute BAT Tests
+= Run a Test Suite
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this procedure, you execute the tests you wrote earlier in the workflow. Using the BAT CLI, you can execute one test at a time or all tests listed in the bat.yaml. By default, BAT generates HTML and JSON reports about the test results.
+Using the BAT CLI, you can execute one test at a time or all tests listed in the `bat.yaml` file for a test suite.
 
-. Run the `bat` command in the directory where bat.yaml is located.
-+
-The output shows the results of the test and location of reports. For example:
-+
-----
-bat
-BAT Version: 1.0.51
-#  File: ./tests/step.0.dwl
+== To Run Tests
 
-    api-mon
-      ✓ GET http://apimon.cloudhub.io/users (295.45ms)
-        ✓ 200 must equal 200
-#  File: ./tests/step.1.dwl
+* To run all of the tests that are listed in the `bat.yaml` file for a test suite, run the `bat` command at the root level in your test-suite directory. Your `bat.yaml` file for the test suite must also be at the root level in this directory.
 
-    api-mon
-
-        gets a list of users
-          ✓ GET http://apimon.cloudhub.io/users (239.88ms)
-            ✓ 200 must equal 200
-            ✓ "application/json" must equal "application/json"
-{
-  kind: "Assertion",
-  pass: true,
-  name: "10 must equal 10",
-  softFail: false,
-  result: {
-    kind: "Assertion::Equal",
-    expectedValue: "10" as String {mimeType: "application/dw"},
-    givenValue: "10" as String {mimeType: "application/dw"}
-  }
-}
-            ✓ 10 must equal 10
-#  Reporter: bat/Reporters/JSON.dwl >> /var/folders/w5/_cc_5_z12pq3qnymf3f0zh840000gq/T/bat_report_20180225145000.json
-#  Reporter: bat/Reporters/HTML.dwl >> /var/folders/w5/_cc_5_z12pq3qnymf3f0zh840000gq/T/bat_report_20180225145000.html
-----
+* To run a single test that is listed in the `bat.yaml` file for a test suite, run the command `bat ./folder/file.dwl` at the root level in your test-suite directory:
 +
-. Open the HTML report shown on the last line of the output above.
+`folder`:: Specify the path to the test file.
 +
-image::bat-html-report.png[]
+`file.dwl`:: Specify the name of the test file.
 +
-. Expand and click items in the navigation pane to drill down.
-+
-Details about the request, response, assertions, and the raw data of the response appear.
-+
-image::bat-html-report-request.png[]
+Your `bat.yaml` file for the test suite must also be at the root level in the test-suite directory.
 
-Next, xref:bat-playground-task.adoc[Run tests in the BAT Playground].
+== What To Do Next
 
-== See Also
+By default in the example test suite that you can generate with the `bat init` command, BAT generates an HTML report about the test results. The configuration of this report is in the `reporters` section on the `bat.yaml` file.
```

Comment:
```suggestion
By default, in the example test suite that you can generate with the `bat init` command, BAT generates an HTML report about the test results. The configuration of this report is in the `reporters` section on the `bat.yaml` file:
```
You could significantly shorten a lot of texts like this one, assuming that your audience is following your narrative flow:
```suggestion
By default, BAT generates an HTML report about the test results. You configure this report in the `reporters` section of the `bat.yaml` file:
```

Reply:
(I think that I might have missed editing for colons ending texts that introduce graphics; please check?)